draft-webtransport-http2.md

```diff
@@ -5,7 +5,7 @@ docname: draft-webtransport-http2-latest
 
```

Comment:
Line 394: "MUST NOT send data" - Should we say DATA frames with non-zero payload?

Reply:
Done